tests/instruments/attocube/test_anc300.py

```diff
@@ -95,17 +98,14 @@ def write(self, command, check_ack=True):
         """
         self.lastcommand = command
         self.connection.write(command + self.write_termination)
-        if check_ack:
-            reply = self.connection.read()
-            msg = reply.strip(self.read_termination)
-            self.check_acknowledgement(msg)
 
 
 @pytest.fixture(autouse=True)
 def modding(monkeypatch):
     monkeypatch.setattr(anc300, "AttocubeConsoleAdapter", Mock_Adapter)
 
 
+@pytest.mark.xfail
```

Comment:
Please add a reason why we expect this to fail currently to the xfail.

Reply:
I don't know the right Protocol.

Reply:
What do you mean? usage for xfail? -> [docs](https://docs.pytest.org/en/7.1.x/how-to/skipping.html#reason-parameter)

Reply:
I did not know, how exactly the device protocol works, therefore I was not able to simulate the right answer of the device.

Reply:
I fixed the Adapter (passing now the test) according to the manual.